CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - unreleased
```

Comment:
since we're not ready for a 1.0.0, I renamed the milestones. It should now be 0.10.0

Reply:
I've updated the projects milestone where the next one was named `0.1.0` :-)